packages/sexplib/sexplib.v0.9.0/opam

```diff
@@ -10,7 +10,7 @@ build: [
 ]
 depends: [
   "base"     {= "v0.9.0"}
-  "jbuilder" {build & >= "1.0+beta4"}
+  "jbuilder" {build & >= "1.0+beta4" & < "1.0+beta12"}
```

Comment:
@trefis @hhugo FYI, this is to handle the removal of `per_file` in beta12.

Reply:
I have the vague feeling that we should never have such upper bound constraints on jbuilder if we don't want to make it a nightmare for people.

Let's say you revert that and I do a minor release of sexplib?

Reply:
We need a minor release of sexplib for 4.06 compatibility anyway :-) But if the build of sexplib.0.9.0 is broken under future jbuilder releases, isn't it right to have a constraint? Your minor release will be sexplib.0.9.1 anyway, right? (Or are you suggesting to add the constraint back, but only after the minor release is out, so that people have a transition path by updating sexplib first?)

Reply:
> We need a minor release of sexplib for 4.06 compatibility anyway :-)

![:|](http://parce-q.eu/l.jpg)

> But if the build of sexplib.0.9.0 is broken under future jbuilder releases, isn't it right to have a constraint? Your minor release will be sexplib.0.9.1 anyway, right? 

That is correct.
I was indeed hoping to release sexplib before this PR gets merged.